hs/CssGen.hs

```diff
@@ -78,6 +77,8 @@ headerCSS = do
                 padding 0 (px 10) 0 (px 10)
                 a <? do
                     color white
+                a # hover <? do
```

Comment:
You can actually use a nested selector here - something like `hover & do`

If you do a search through this file for `&`, you'll see how it's used.